css/index_style.css

```diff
@@ -0,0 +1,206 @@
+main section a {
+  width: max-content;
+}
+
+.hero img {
+  width: 100%;
+}
+
+.explore {
+  padding-bottom: 33px;
+}
+
+.discount {
+  background-color: var(--lilac);
+  margin-top: -5px;
+}
+
+.feature-img {
+  width: 70px;
+}
+.feature-img img {
+  width: 100%;
+}
+.vision {
+  background-color: var(--light-Green);
+}
+.vision .flex_col {
+  align-items: center;
+  text-align: center;
+}
+.vision img,
+.wellness_path img,
+.discover img {
+  width: 100%;
+}
+
+.wellness_path {
+  background-color: var(--lilac);
+  padding-left: 30px;
+  padding-right: 30px;
+}
+.wellness_selection {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-areas:
+    "coldpr coldpr coldpr"
+    "smooth smooth smooth"
+    "shot shot shot"
+    "button button button";
+}
+.wellness_coldpr {
+  grid-area: coldpr;
+}
+.wellness_smoothie {
+  grid-area: smooth;
+}
+.wellness_shots {
+  grid-area: shot;
+}
+.wellness_selection a {
+  grid-area: button;
+  justify-self: center;
+}
+.get_juice {
+  background-color: var(--light-Green);
+}
+.circle_border {
+  width: 44px;
+  height: 44px;
+  border: solid 1px var(--black);
+  border-radius: 50%;
+}
+.follow {
+  display: grid;
+  grid-template-rows: auto 1fr 1fr;
+  grid-template-columns: repeat(12, 1fr);
+}
+
+.follow_green {
+  background-color: var(--dark-Green);
+  grid-row: 3 / 4;
+  grid-column: 1 / -1;
+}
+.img_apples {
+  align-self: flex-end;
+  grid-row: 3 / 4;
+  grid-column: 5 / 9;
+}
+.img_papaya {
+  align-self: center;
+  grid-row: 2 / 4;
+  grid-column: 3 / 7;
+}
+
+.img_bag {
+  align-self: flex-start;
+  grid-row: 1 / 3;
+  grid-column: 4 / 8;
+}
+
+.img_kale {
+  align-self: center;
+  grid-row: 2 / 4;
+  grid-column: 7 / 11;
+}
+
+.img_apples img {
+  transform: rotate(333.85071717456117deg);
+  width: 100%;
+}
+.img_papaya img {
+  transform: rotate(16.69666345366653deg);
+  width: 100%;
+}
+.img_bag img {
+  transform: rotate(347.34482520820825deg);
+  width: 100%;
+}
+.img_kale img {
+  transform: rotate(17.142845342601447deg);
+  width: 100%;
+}
+.follow img {
+  width: 100%;
+}
+@media screen and (min-width: 481px) {
+  .explore {
+    padding: 65px 0;
+  }
+  .explore .font_2 {
+    font-size: 24px;
+  }
+  .discount .font_3 {
+    font-size: 22px;
+  }
+  .features {
+    text-align: left;
+    display: flex;
+    justify-content: space-between;
+  }
+  .feature-img {
+    width: 101px;
+    height: 101px;
+  }
+  .feature {
+    width: 236px;
+    margin: 0 20px;
+    justify-content: start;
+    align-items: flex-start;
+  }
+  .vision {
+    display: flex;
+    padding-top: 70px;
+    justify-content: center;
+  }
+  .vision div {
+    width: 50%;
+  }
+  .vision .flex_col {
+    align-items: flex-start;
+    text-align: left;
+    width: 490px;
+  }
+  .vision img {
+    width: 557px;
```

Comment:
You're setting quite a few fixed widths which I wouldn't recommend. These mean your layouts will look perfect at a very particular screen size, but won't adapt to for example a slightly larger phone, or a slightly smaller desktop. 
It's best to set widths either in `%` or in `fr` if using grid. This ensures you are setting proportions instead of fixed dimensions. In the case of this image, it's basically taking up half of the screen in desktop view, so that's what you should be aiming for rather than a fixed width.